public/main.css

```diff
@@ -33,9 +35,13 @@ header {
   padding: 20px;
 }
 header .content {
+  display: -webkit-flex;
+  display: -ms-flexbox;
   display: flex;
 }
 header .spacer {
+  -webkit-flex: 1;
+  -ms-flex: 1;
   flex: 1;
 }
 header a {
```

Comment:
Because every time you run 'gulp' right now, '/public' gets updated with changes, which you'd have to manually reset before each commit.